content/en/cosign/keyless.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Keyless Signatures"
+category: "Cosign"
+position: 122
+---
+
+The full design document for this can be found [here](https://docs.google.com/document/d/1461lQUoVqbhCve7PuKNf-2_NfpjrzBG5tFohuMVTuK4/edit#).
+
+This document explains how the `keyless` signatures work in `cosign`.
+Try it out!
+
+This signature mode relies on the Sigstore Public Good Instance, which is rapidly heading toward a GA release!
+We don't have a date yet, but follow along on the [GitHub project](https://github.com/orgs/sigstore/projects/5).
+
+The following examples use this image:
+
+```shell
+$ IMAGE=gcr.io/dlorenc-vmtest2/demo
```

Comment:
Ideally we would have a more generic image but it should be consistent throughout so we don't need this to be a blocker

Reply:
Yep I thought the same, but realised that most docs have snippets similar to this

Reply:
Maybe something public like gcr.io/google-containers/busybox?

Reply:
Is there another image you'd like me to put here? Possibly
```shell
$ IMAGE=busybox:1
$ IMAGE_DIGEST=$IMAGE@sha256:907ca53d7e2947e849b839b1cd258c98fd3916c60f2e6e70c30edbf741ab6754
```

Although I personally for me to run the keyless example had to do some stuff with pushing it to `ttl.sh` though to ensure I could sign it. Do we have an official `cosign` demo gcr.io example we can put here? If so, I won't have the permissions to push the signature.

Reply:
I'm partial to the way the quick start in the Cosign README works: https://github.com/sigstore/cosign#quick-start

Basically, we set up a few env vars at the beginning using ttl.sh, then use *those* throughout for examples. That way it's easy to substitute your own (which is what people will do in practice) but you can test all the README examples.